.github/workflows/ci.yml

```diff
@@ -17,186 +17,186 @@ on:
     types: [opened, synchronize]
 
 env:
-  flutter_version: '2.8.x'
-  flutter_channel: 'stable'
-  java_version: '12.x'
+  flutter_version: "2.8.x"
+  flutter_channel: "stable"
+  java_version: "12.x"
 
 jobs:
   analyze:
     runs-on: ubuntu-latest
     timeout-minutes: 5
     steps:
-    - name: Fetch code from repository
-      uses: actions/checkout@v2
-    - name: Cache Flutter install
-      uses: actions/cache@v2
-      with:
-        path: /opt/hostedtoolcache/flutter
-        key: cache-flutter-${{ env.flutter_channel }}-${{ env.flutter_version }}-${{ runner.os }}
-    - name: Setup the Flutter environment
-      uses: subosito/flutter-action@v1
-      with:
-        channel: ${{ env.flutter_channel }}
-        flutter-version: ${{ env.flutter_version }}
-    - name: Get Flutter dependencies
-      run: flutter pub get .
-    - name: Check for any formatting issues
-      run: flutter format --dry-run --set-exit-if-changed .
-    - name: Build generated code
-        run:   |
+      - name: Fetch code from repository
+        uses: actions/checkout@v2
+      - name: Cache Flutter install
+        uses: actions/cache@v2
+        with:
+          path: /opt/hostedtoolcache/flutter
+          key: cache-flutter-${{ env.flutter_channel }}-${{ env.flutter_version }}-${{ runner.os }}
+      - name: Setup the Flutter environment
+        uses: subosito/flutter-action@v1
+        with:
+          channel: ${{ env.flutter_channel }}
+          flutter-version: ${{ env.flutter_version }}
+      - name: Get Flutter dependencies
+        run: flutter pub get .
+      - name: Check for any formatting issues
+        run: flutter format --dry-run --set-exit-if-changed .
+      - name: Build generated code
+        run: |
           flutter pub run flutter_oss_licenses:generate.dart lib/presentation/licenses/oss_licenses.dart
```

Comment:
https://github.com/CollActionteam/collaction_app/runs/4767976764?check_suite_focus=true#step:8:193

Looks like we need to call `flutter format .` again after this line.